data/en/products.yml

```diff
@@ -0,0 +1,15 @@
+############################# Products ###################################
+products:
+  enable: true
+  title: Products
+  subtitle: With deep industry and government experience, we can deliver on your most important priorities.
+  products:
+    - product_name: Ensign
+      description: Ensign is a database-meets-event-streaming platform for data science teams that want to get their data moving and their models into production.
```

Comment:
The description text for each product is all placeholder text.

Reply:
Looks good to me!